test/functional/specs/C14400.js

```diff
@@ -0,0 +1,77 @@
+import { t, ClientFunction } from "testcafe";
+import createNetworkLogger from "../helpers/networkLogger";
+import getResponseBody from "../helpers/networkLogger/getResponseBody";
+import { responseStatus } from "../helpers/assertions";
+import fixtureFactory from "../helpers/fixtureFactory";
+import createResponse from "../../../src/core/createResponse";
+import generalConstants from "../helpers/constants/general";
+
+const networkLogger = createNetworkLogger();
+const { ecidRegex } = generalConstants;
+
+fixtureFactory({
+  title:
+    "C14400: When ID migration is disabled and no identity cookie is found but legacy s_ecid cookie is found, the ECID should not sent on the request",
+  requestHooks: [networkLogger.edgeEndpointLogs]
+});
+
+test.meta({
+  ID: "C14400",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+const apiCalls = ClientFunction(() => {
+  document.cookie = "s_ecid=MCMID%7C16908443662402872073525706953453086963";
+
+  window.alloy("configure", {
+    configId: "9999999",
+    orgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
+    edgeBasePath: window.edgeBasePath,
+    debugEnabled: true,
+    idMigrationEnabled: false
+  });
+
+  return window.alloy("event", {
+    viewStart: true,
+    xdm: {
+      device: {
+        screenHeight: 1
+      }
+    }
```

Comment:
Remove the `xdm` block if it's unnecessary for the test. Same with the other tests.